livre/core/render/Renderer.cpp

```diff
@@ -35,12 +35,19 @@ Renderer::~Renderer()
 void Renderer::render( const Frustum& frustum,
                        const ClipPlanes& planes,
                        const PixelViewport& view,
-                       const NodeIds& bricks )
+                       const NodeIds& bricks,
+                       const uint8_t renderStages )
```

Comment:
uint8_t is a premature optimization. The flags can be defined as static const ints.